python/ray/experimental/channel/torch_tensor_nccl_channel.py

```diff
@@ -553,9 +554,10 @@ def _do_init_nccl_group(
 ):
     import torch
 
-    assert (
-        ray.get_gpu_ids()
-    ), "Actors participating in NCCL group must have at least one GPU assigned"
+    if not custom_nccl_group or not isinstance(custom_nccl_group, CPUNcclGroup):
```

Comment:
```suggestion
    if not (custom_nccl_group and isinstance(custom_nccl_group, CPUNcclGroup)):
```
nit, a bit more readable with a single not.